lib/TraackrApi/TraackrApiObject.php

```diff
@@ -9,6 +9,25 @@
 
    private $curl;
 
+   // Headers passed with each request
+   private $curl_headers  = array(
```

Comment:
+1, also are there headers that we don't want to allow?  Should we have a whitelisted list of headers that can be set?  Thinking about how we altered the Twitter headers in order to bypass a security restriction, is there potential here to do the same?